ssl-cert.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I don't think we store certificates in version control. How did you generate this certificate and what hosts does it trust? We can add setting up certificates as a step in the README instead of doing it for users IMO. For development, one can use tools like https://github.com/FiloSottile/mkcert to create SSL certs.